docs/pages-for-subheaders/azure-marketplace-payg-integration.md

```diff
@@ -0,0 +1,203 @@
+
+**My Azure account is in the USA, but I want to deploy Rancher in another Azure region, a region that is in a country where I currently cannot transact Rancher Prime, is this possible?**
+
+Yes, as long as your Azure account is billed to one of the allowed countries, it is possible to deploy Rancher Prime in any Azure region.
+
+**Is this listing available in China?**
+
+While it is not possible to transact/bill Rancher Prime in China, it is possible to deploy into Azure regions in China.
+
+### Billing
+
+**I have an existing Rancher Prime subscription; can I use this on Azure?**
+
+Self-installed BYOS (Bring Your Own Subscription) Rancher Prime deployments are supported on Azure; however, billing will not be via the Azure Marketplace. Once the existing subscription term ends, you can purchase Rancher Prime via the Azure Marketplace and reconfigure your cluster to support monthly billing via Azure.
+
+**I have an existing deployment covered by a Rancher subscription; can I use this new listing in the Azure Marketplace for new deployments?**
+
+Yes, the listing works independently from your existing Rancher Prime subscriptions. Only deployments through the marketplace listing will be billed through Azure. Support is always direct from SUSE.
+
+**Tell me more about how the billing for Rancher Prime works via Azure?**
+
+When purchasing Rancher Prime via the Azure Marketplace, the billing is as follows:
+
+- Billing is monthly and handled via Azure.
+- Managed nodes are counted hourly when Rancher is active and added to a usage total.
+- An average node count is calculated for the month.
+- There is a monthly usage charge for each node in the average node count.
+- The monthly usage charge depends on the number of nodes in use.
+- There is a 5-node minimum, if the average node count is less than 5 nodes, the charge will be for 5 nodes.
+
+**What are the pricing tiers?**
+
+Rancher Prime has different pricing tiers when purchasing via the Azure Marketplace. This is based on the number of nodes which Rancher is managing.Details of the tiers are shown below, please check the listing for further pricing information.
+
+**Is there a way to try Rancher before purchasing?**
+
+If using the Rancher Prime listing in the Azure Marketplace, billing will commence from the time of deployment. You can try Rancher by deploying it per standard documentation. When ready to benefit from a supported platform and have this billed through Azure, deploy Rancher Prime via the Azure Marketplace and migrate your configuration.
+
+**How does SUSE calculate the ‘average number of managed nodes’ to bill for?**
+
+The average node count is calculated by adding the number of managed nodes (counted hourly) and dividing by the number of hours Rancher has been active in the billing cycle. Three examples are shown in the table below.
+
+:::note
+In our example month, we are using 730 hours; this may differ depending on the number of days in the month and the billing cycle.
```

Comment:
I don't understand the examples below...
we bill the number of nodes rather than the number of hours, so I would expect that the first example would get billed for 10 nodes, the second for 15 (maybe? @guangyee can you confirm how the averaging would work in this example?), and the third for 20...  Maybe I'm dense and need Guang , Charu and Meera to explain it to me :)

Reply:
I saw this in the rendered version and it seems correct, I missed the table aspect of it when reading just the markdown